.github/workflows/ReusableTest.yml

```diff
@@ -0,0 +1,47 @@
+name: Reusable test
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        type: string
+        default: "1"
+      os:
+        required: false
+        type: string
+        default: ubuntu-latest
+      arch:
+        required: false
+        type: string
+        default: x64
+      allow_failure:
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  test:
+    name: Julia ${{ inputs.version }} - ${{ inputs.os }} - ${{ inputs.arch }} - ${{ github.event_name }}
+    runs-on: ${{ inputs.os }}
+    continue-on-error: ${{ inputs.allow_failure }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "GitHub Action"
```

Comment:
What's this for?

Reply:
Part of #122. This started being necessary for some unknown reason. In retrospect, it could be related to the read-only issue.

Reply:
Since this is the only comment and it is related to a different issue, I will continue with the PR merge. Please let me know if you have any issues with this so we create an issue.